src/Browser/Json.php

```diff
@@ -33,6 +48,63 @@ public function assertMatches(string $expression, $expected): self
         return $this;
     }
 
+    /**
+     * @param string $selector JMESPath selector
+     */
+    public function assertHas(string $selector): self
+    {
+        Assert::try(fn() => $this->search("length({$selector})"));
+
+        Assert::pass();
+
+        return $this;
+    }
+
+    /**
+     * @param string $selector JMESPath selector
+     */
+    public function assertMissing(string $selector): self
+    {
+        try {
+            $this->search("length({$selector})");
+        } catch (\RuntimeException $e) {
+            Assert::pass();
+
+            return $this;
+        }
+
+        Assert::fail("Selector \"{$selector}\" does exists.");
+    }
+
+    /**
+     * @phpstan-param callable(Json $json): void $assert
```

Comment:
```suggestion
     * @param callable(Json):void $assert
```

Should work ok and not bother php-cs-fixer (it doesn't like the space).